docs/getting-started.md

```diff
@@ -3,6 +3,48 @@ sidebar_position: 1
 slug: /
 ---
 
-# Getting Started
+## How to Build a Guided Project
 
-Welcome to the **Author Workbench Knowledgebase**. We are in the process of building this knowledgebase to help you build **amazing courses and guided projects** with Author Workbench.
+Congratulations on creating your Guided Project. This article will walk you through the steps to create and publish a Guided Project.
+
+## Prerequisites
+
+If you haven't went through our onboarding process, please visit https://www.skills.network/authoring/ first and follow the instructions.
+
+## Get Started
+
+Upon creating a project, you should see the following screen:
+<img width="1904" alt="guided project creation" src="https://user-images.githubusercontent.com/46271636/171186797-e3a65a85-99a4-4bc6-9be0-ee30a5004e46.png">
```

Comment:
fixes syntax error, app does not run without this change
```suggestion
<img width="1904" alt="guided project creation" src="https://user-images.githubusercontent.com/46271636/171186797-e3a65a85-99a4-4bc6-9be0-ee30a5004e46.png" />
```

These screenshots are also pretty old. The UI has changed a decent amount and will change even more this week with a new update. I think all screenshots will have to be replaced with new ones so it's easy for people to follow.

I'll ping you on Slack when the new UI is up (hopefully later today)